modules/dataframe/include/inviwo/dataframe/io/csvreader.h

```diff
@@ -89,6 +89,14 @@ class IVW_MODULE_DATAFRAME_API CSVReader : public DataReaderType<DataFrame> {
     CSVReader& setNumberOfExampleRows(size_t rows);
     size_t getNumberOfExamplesRows() const;
 
+    /** @see CSVReader::defaultRowComment */
+    CSVReader& setRowComment(std::string_view comment);
+    const std::string& getRowComment() const;
+
+    /** @see CSVReader::defaultKeepOnly */
```

Comment:
this comment doesn't explain what it is doing/what the purpose is. The member only refers back to this function.